.chloggen/remove_access_token_passthrough.yaml

```diff
@@ -0,0 +1,14 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
@dloucasfx could we just add `include_metadata: true` by default in the sfx recv in gateway so this is non-breaking?